app/components/node-navbar/link/component.ts

```diff
@@ -0,0 +1,32 @@
+import { className, tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+
+import Component from '@ember/component';
+
+@tagName('li')
+export default class NodeNavbarLink extends Component {
+    @service router;
+
+    id: 'string';
+    useLinkTo?: boolean;
```

Comment:
Minor: might be nice to default this to `true`, so as we convert we can just delete the `useLinkTo=false`s and eventually remove this attr altogether.